### ISSUE TITLE
Investigate delay on "Stop Race" button when database has data

### ISSUE THREAD

**Issue by HazardCreative** [RotorHazard/RotorHazard#17]
Multiple users report a significant delay during 'stop race' when there are more than a few pilots and saved races. Is there an issue here that needs resolved?

**Comment by HazardCreative** [comment]
I understand that about saving the laps, but this delay is also reported on
simply stopping a race. There's almost no processing happening then.

I ran into this at my least event but figured it was the local wifi until I
heard about report. I tested the very same database at home and didn't have
the issue locally. Maybe it's related to uptime? Need to be able to
reproduce.

**Comment by MoonKIE666** [comment]
I also thought that these time delays were due to my tablet pc. But now that I'm reading you, the same thing happens to me.

My DELTA5 is eight nodes, although I have two deactivated nodes because we usually fly 6. At first everything works fine and I can move through the menus in an agile way and without time delay.

But to which we took about 5 rounds, the system starts to slow down significantly. It almost always stops when I hit the STOP button or when I hit the SAVE LAPS button. It takes more than fifteen seconds, and although it gives you to refresh the web page, it is left thinking.
